app/components/HtmlDocument/index.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/no-danger */
+
+import React, { PropTypes } from 'react';
+import htmlescape from 'htmlescape';
+
+// We use this component only on the server side.
+export default function HtmlDocument({ lang, head, css, appMarkup, state, assets, webpackDllNames }) {
+  const attrs = head.htmlAttributes.toComponent();
+  return (
+    <html lang={lang} {...attrs}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        {/* Allow installing the app to the homescreen */}
+        <link rel="manifest" href="/manifest.json" />
+        <meta name="mobile-web-app-capable" content="yes" />
+
+        {head.title.toComponent()}
+        {head.meta.toComponent()}
+
+        {/* vendor.css */}
+        <link href={assets.main.css} rel="stylesheet" />
+
+        {/* app css */}
+        <style type="text/css" dangerouslySetInnerHTML={{ __html: css }} />
+      </head>
+      <body>
+        {/* Display a message if JS has been disabled on the browser. */}
+        <noscript>
+          If you are seeing this message, that means <strong>JavaScript has been disabled on your browser</strong>
+          , please <strong>enable JS</strong> to make this app work.
+        </noscript>
```

Comment:
Let's change this message, since if JS is disabled one can actually see things now. This should be a popup or something saying that this site will work better with JS enabled.